samples/sensor/ina219/src/main.c

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2021 Leonard Pollak
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <stdio.h>
+#include <drivers/sensor.h>
+
+
+#if !DT_HAS_COMPAT_STATUS_OKAY(ti_ina219)
+#error "No ti,ina219 compatible node found in the device tree"
+#endif
```

Comment:
better use `DEVICE_DT_GET_ONE` so that you get a compilation error if there's no device.

Reply:
updated.